### ISSUE TITLE
No support for `staxids` field with DAA?

### ISSUE THREAD

**Issue by standage** [bbuchfink/diamond#489]
I built a custom Diamond database with taxonomic information included. If I search against this database and produce tabular output, I have no problem adding `staxids` to `--outfmt`.

```
diamond blastp \
    --query mystery-seqs.faa \
    --db mydb.dmnd \
    --out results.tsv \
    --threads 4 \
    --outfmt 6 qseqid sseqid pident length qstart qend qlen sstart send slen evalue nident stitle staxids
```

If I try to store results in a DAA file and then use `diamond view` to convert to tabular...

```
diamond blastp \
    --query mystery-seqs.faa \
    --db mydb.dmnd \
    --out results.daa \
    --outfmt 100 \
    --threads 4
diamond view \
    --daa results.daa \
    --out results.tsv \
    --outfmt 6 qseqid sseqid pident length qstart qend qlen sstart send slen evalue nident stitle staxids
```

...I get the following error.

```
diamond v2.0.11.149 (C) Max Planck Society for the Advancement of Science
Documentation, support and updates available at http://www.diamondsearch.org
Please cite: http://dx.doi.org/10.1038/s41592-021-01101-x Nature Methods (2021)

#CPU threads: 4
Loading subject IDs...  [0.001s]
Scoring parameters: (Matrix=blosum62 Lambda=0.267 K=0.041 Penalties=11/1)
DB sequences = 4747
DB sequences used = 213
DB letters = 1413209
Error: Taxonomy features are not supported for the DAA format.
```

Is this the intended behavior?

**Comment by bbuchfink** [comment]
Yes, the daa format does not support taxonomy features. It is also considered a legacy format at this time, so there are no plans for implementing this.

**Comment by standage** [comment]
Thanks for the response!

> It is also considered a legacy format at this time

Good to know. I assume you're not replacing DAA with an alternative archive format, correct? With BLAST, I use `.asn` archives and the `blast_formatter` command extensively to store alignment results once and format the output in numerous ways. Is support for this type of behavior with Diamond going to be dropped in the future?

**Comment by bbuchfink** [comment]
I plan to make it possible reading in and reformatting alignment files in tsv format. The time frame for this should be 2-3 months. I also don't intend to remove support for the daa format.

**Comment by standage** [comment]
With the planned features, will it be possible to reformat Diamond output to get both "BLAST pairwise" and tabular with staxids? (Thank you for your responses and your patience with my questions!)

**Comment by bbuchfink** [comment]
Yes, that should both be possible.

**Comment by standage** [comment]
Wonderful. Thanks for your excellent work on this tool!
